src/Policies/ImagePolicy.php

```diff
@@ -14,36 +14,21 @@ class ImagePolicy
 
     public function viewAny(UserInterface $user)
     {
-        return $user->hasPermissionTo('view images') ? true : false;
+        return $user->hasPermissionTo('view images');
     }
 
     public function view(UserInterface $user, Image $image)
     {
-        return $user->hasPermissionTo('view images') ? true : false;
+        return $user->hasPermissionTo('view images');
     }
 
     public function create(UserInterface $user)
     {
-        return $user->hasPermissionTo('create images') ? true : false;
+        return $user->hasPermissionTo('create images');
     }
 
     public function update(UserInterface $user, Image $image)
     {
-        return $user->hasPermissionTo('update images') ? true : false;
-    }
-
-    public function delete(UserInterface $user, Image $image)
-    {
-        return false;
-    }
-
-    public function restore(UserInterface $user, Image $image)
-    {
-        return false;
-    }
-
-    public function forceDelete(UserInterface $user, Image $image)
-    {
-        return false;
```

Comment:
I like leaving these here so it is explicit that users cannot delete the resource.